### ISSUE TITLE
Settings are reset on quit

### ISSUE THREAD

**Issue by Finesse** [AdguardTeam/AdGuardForSafari#107]
### Steps to reproduce

1. Launch the application or open preferences from the icon dropdown.
2. Turn the «Show AdGuard for Safari in the menu bar» and the «Use optimized filters» settings off.
3. Quit the application by pressing ⌘Q.
4. Start the application again and open the settings screen.

### Expected behavior

The «Show AdGuard for Safari in the menu bar» and the «Use optimized filters» settings are **off**.

### Actual behavior

The «Show AdGuard for Safari in the menu bar» and the «Use optimized filters» settings are **on**.

### Your environment

* AdGuard for Safari version: 1.2.0
* Environment name and version: Safari 12.0.2 (14606.3.4)
* Operating System: macOS 10.14.2 (18C54)
* Filters you use in AdGuard: the default (AdGuard Base filter, AdGuard Spyware filter, AdGuard Safari filter)
* Any other browser extensions you have installed: none

**Comment by ikir83** [comment]
yes settings keeps get resetting, I need to remove the menu icon every time.

**Comment by Finesse** [comment]
Fixed, thank you!
